kwonhayeon/likelion-assignment-crud/src/main/java/com/example/likelionassignmentcrud/feedback/api/application/FeedbackService.java

```diff
@@ -0,0 +1,61 @@
+package com.example.likelionassignmentcrud.feedback.api.application;
+
+import com.example.likelionassignmentcrud.feedback.api.dto.request.FeedbackSaveRequestDto;
+import com.example.likelionassignmentcrud.feedback.api.dto.response.FeedbackInfoResponseDto;
+import com.example.likelionassignmentcrud.feedback.api.dto.response.FeedbackListResponseDto;
+import com.example.likelionassignmentcrud.feedback.api.domain.model.Feedback;
+import com.example.likelionassignmentcrud.feedback.api.domain.repository.FeedbackRepository;
+import com.example.likelionassignmentcrud.session.domain.model.Session;
+import com.example.likelionassignmentcrud.session.domain.repository.SessionRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class FeedbackService {
+
+    private final SessionRepository sessionRepository;
+    private final FeedbackRepository feedbackRepository;
+
+    // 피드백 저장
+    @Transactional
+    public void feedbackSave(FeedbackSaveRequestDto dto) {
+        Session session = sessionRepository.findById(dto.sessionId())
+                .orElseThrow(IllegalArgumentException::new);
```

Comment:
단순히 IllegalArgumentException을 반환하게 되면 사용자에게 어떤 문제가 발생했는지 알기 어려워요 😢😢
커스텀 예외를 만들어서 메시지와 함께 던지는 형식 등 더욱 직관적인 방식을 생각해보면 좋을 것 같아요~

```suggestion
                .orElseThrow(() -> new FeedbackNotFoundException("해당 세션을 찾을 수 없습니다."));
```